commerce_coordinator/apps/commercetools/utils.py

```diff
@@ -98,11 +99,12 @@ def send_fulfillment_error_email(
         logger.exception(f"Encountered exception sending Fulfillment error email. Exception: {exc}")
 
 
-def format_amount_for_braze_canvas(centAmount):
+def format_amount_for_braze_canvas(cent_amount, currency_code):
     """
     Utility to convert amount to dollar with 2 decimals percision. Also adds the Dollar signs to resulting value.
     """
-    return f"${(centAmount / 100):.2f}"
+    formatted_price = format_currency(cent_amount / 100, currency_code, locale='en_US')
```

Comment:
Should there not be a space between the symbol/code and the amount?

Reply:
please see latest changes.